src/codeu/chat/client/commandline/Chat.java

```diff
@@ -46,11 +46,19 @@ public Chat(Context context) {
   // the system wants to exit, the function will return false.
```

Comment:
I think that you need some more import statements at the top of the file for this to work.